### PR TITLE
12.0 afc 2071 Final report fixes

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

File: outplacement_final_report/i18n/sv_SE.po [hunk 1]
```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 12.0-20210129\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-03-24 09:10+0000\n"
-"PO-Revision-Date: 2021-03-24 10:44+0100\n"
+"POT-Creation-Date: 2021-04-13 14:47+0000\n"
+"PO-Revision-Date: 2021-04-13 16:49+0200\n"
 "Last-Translator: <>\n"
```
Comment (nils-vertel) [inline outplacement_final_report/i18n/sv_SE.po:11]: Set last translator to your self?

File: outplacement_final_report/__manifest__.py [hunk 2]
```diff
@@ -10,7 +10,12 @@
     'author': 'Vertel AB',
```
Comment (nils-vertel) [inline outplacement_final_report/__manifest__.py:12]: Update version and backlog

**Comment by fredrikarvas-vertel** [comment]
Please add version bump and description

File: outplacement_final_report/models/outplacement.py [hunk 3]
```diff
@@ -53,34 +53,57 @@ class OutplacementGoal(models.Model):
...
     @api.constrains('free_text')
     @api.one
     def _constrain_free_text(self):
-        if self.free_text and len(self.free_text) > 2000:
-            raise ValidationError(_('Number of characters must not exceed 2000'))
-
-    @api.constrains('other_motivation')
+        if self.free_text and len(self.free_text) > 100:
+            raise ValidationError(_('Number of characters in the free text must not exceed 100'))
+
```
Comment (nils-vertel) [inline outplacement_final_report/models/outplacement.py]: Extra spaces?

File: outplacement_final_report/views/outplacement_view.xml [hunk 4]
```diff
@@ -86,22 +102,97 @@
...
+
+    <menuitem id="goal_configuration" parent="outplacement.outplacement_configuration" name="Goals"
+              groups="outplacement.group_outplacement_admin"
+              sequence="90" action="outplacement_goal_action"/>
+    <menuitem id="goal_step_configuration" parent="outplacement.outplacement_configuration" name="Steps"
+              groups="outplacement.group_outplacement_admin"
+              sequence="91" action="outplacement_goal_step_action_tree"/>
+
+
 </odoo>
```
Comment (nils-vertel) [inline outplacement_final_report/views/outplacement_view.xml]: Missing line at end of file

File: outplacement_final_report_send/models/outplacement.py [hunk 5]
```diff
@@ -10,10 +12,18 @@ class Outplacement(models.Model):
 
     def send_final_report(self):
         client_config = self.env['ipf.final_report.client.config'].search([], limit=1)
+        self.fr_send_date = "" #set to datetime.now with correct formating and timezone
```
Comment (nils-vertel) [inline outplacement_final_report_send/models/outplacement.py]: This looks wrong

**Review by nils-vertel — CHANGES_REQUESTED** [review]

**Pull request merged**